zips/zip-0225.rst

```diff
@@ -145,15 +145,15 @@ Transaction Format
 |``1``                        |``flagsOrchard``          |``byte``                                |An 8-bit value representing a set of flags. Ordered from LSB to MSB: |
 |                             |                          |                                        | * ``enableSpendsOrchard``                                           |
 |                             |                          |                                        | * ``enableOutputsOrchard``                                          |
-|                             |                          |                                        | * The remaining bits are set to ``0``.                              |
+|                             |                          |                                        | * The remaining bits are set to :math:`0\!`.                        |
```

Comment:
Why are these still using `:math:` instead of `$...$`?

Reply:
Because it's a table with column-sensitive markup. `$0$.` would be replaced by ```:math:`0\!`.``` which would throw off the position of the `|` ending the column. It is possible to compensate for that manually but it's easier to just use `:math:`.